### PR TITLE
Fix config path handling

### PR DESCRIPTION
**Pull request by ishitatsuyuki**
So it's usable as it is described.

### CODE REVIEW

File: src/options.rs [hunk 1]
```diff
@@ -63,6 +63,7 @@ impl Options {
                 .author(crate_authors!("\n"))
                 .about("A cargo subcommand for checking and applying updates to installed executables")
                 .args(&[Arg::from_usage("-c --cargo-dir=[CARGO_DIR] 'The cargo home directory. Default: $CARGO_HOME or $HOME/.cargo'")
+                            .visible_alias("root")
```
Comment (nabijaczleweli) [inline src/options.rs]: Leave this commit in, this change is 👍

File: src/options.rs [hunk 2]
```diff
@@ -122,7 +123,7 @@ impl ConfigOptions {
         let cdir = cargo_dir();
         ConfigOptions {
             crates_file: match matches.value_of("cargo-dir") {
-                Some(dirs) => (dirs.to_string(), fs::canonicalize(dirs).unwrap()),
+                Some(dir) => (format!("{}/.crates.toml", dir), fs::canonicalize(dir).unwrap().join(".crates.toml")),
```
Comment (nabijaczleweli) [inline src/options.rs]: Nuke this one though, since you broke it.

Comment (ishitatsuyuki) [inline src/options.rs]: Good, saying "directory" and taking a file as argument? You're a super genius.

Comment (nabijaczleweli) [inline src/options.rs]: *What?* Look at `options.rs`#L36-37. Then at `cargo-install-update.md`#L55-59. Then at `cargo-install-update-config.md`#L45-49. Then at `main.rs`#L33. Then at `ops/mod.rs`#L204-232. And maybe, if you're not a bloody squid, you'll connect the blithering dots on this and the fact that **everything is internally and externally consistent** ***and*** **you broke just about the main composite of the library with this shitty fucking commit** will dawn on you.

Comment (ishitatsuyuki) [inline src/options.rs]: What the fuck are you talking about? Try it and see ™

I simply corrected the things to make the command line options work as intended. The comment you mentioned just look like copy-paste, and whoever would mark a variable named "..._file" as directory? Your code below also yields filename, by the way.

**Comment by nabijaczleweli** [comment]
Released in [`v0.8.2`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v0.8.2).

**Pull request merged**